kyo-core/shared/src/main/scala/kyo/aborts.scala

```diff
@@ -47,23 +41,25 @@ object Aborts:
             Tag[Aborts[V]],
             ClassTag[V]
         ): Either[V, T] < (S & VR) =
-            handle(handler, v).asInstanceOf[Either[V, T] < (S & VR)]
+            self.handle(handler)((), v).asInstanceOf[Either[V, T] < (S & VR)]
```

Comment:
I couldn't find a good way to avoid having to pass `()` when a handler doesn't have state.